src/has.ts

```diff
@@ -3,6 +3,60 @@ import { Hash } from './interfaces';
 export const cache: Hash<any> = Object.create(null);
 const testFunctions: Hash<() => any> = Object.create(null);
 
+export const dynamic: boolean = true;
```

Comment:
Shoot I must have used the same branch

On Fri, Jul 17, 2015 at 3:32 PM Bryan Forbes <notifications@github.com>
wrote:

> In src/has.ts <https://github.com/dojo/core/pull/55#discussion_r34923357>:
>
> > @@ -3,6 +3,60 @@ import { Hash } from './interfaces';
> >  export const cache: Hash<any> = Object.create(null);
> >  const testFunctions: Hash<() => any> = Object.create(null);
> >
> > +export const dynamic: boolean = true;
>
> Why are these changes included in the pull request?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dojo/core/pull/55/files#r34923357>.
>